qldbdriver/transaction.go

```diff
@@ -107,20 +107,26 @@ type transactionExecutor struct {
 }
 
 // Execute a statement with any parameters within this transaction.
-func (executor *transactionExecutor) Execute(statement string, parameters ...interface{}) (*Result, error) {
+func (executor *transactionExecutor) Execute(statement string, parameters ...interface{}) (Result, error) {
 	return executor.txn.execute(executor.ctx, statement, parameters...)
 }
 
-// Buffer a Result into a BufferedResult to use outside the context of this transaction.
```

Comment:
This docstring doesn't need to change, as Result and BufferedResult are (or should now be) the interfaces which are consumed and returned.

Reply:
Changed comment back.